Dockerfile

```diff
@@ -2,4 +2,6 @@ FROM quay.io/prometheus/busybox:latest
 
 ADD .build/linux-amd64/operator /bin/operator
 
+USER nobody
```

Comment:
This might cause problems due to https://github.com/kubernetes/kubernetes/pull/56503 You might want to specify a UID here instead.

Reply:
Good point @mrueg, thanks!

@brancz What do you think?